packages/lesswrong/components/common/Home2.jsx

```diff
@@ -4,49 +4,25 @@ import React, { PureComponent } from 'react';
 import withUser from '../common/withUser';
 import { SplitComponent } from 'meteor/vulcan:routing';
 import Users from 'meteor/vulcan:users';
-import AddBoxIcon from '@material-ui/icons/AddBox';
 
 class Home2 extends PureComponent {
-  state = { showShortformFeed: false }
-
-  toggleShortformFeed = () => {
-    this.setState(prevState => ({showShortformFeed: !prevState.showShortformFeed}))
-  }
 
   render () {
     const { currentUser } = this.props
-    const { showShortformFeed } = this.state
 
-    const { SingleColumnSection, SectionTitle, RecentDiscussionThreadsList,
-      HomeLatestPosts, TabNavigationMenu, RecommendationsAndCurated, SectionButton,
-      ShortformSubmitForm } = Components
+    const { RecentDiscussionThreadsList, HomeLatestPosts, TabNavigationMenu, RecommendationsAndCurated } = Components
 
     const shouldRenderSidebar = Users.canDo(currentUser, 'posts.moderate.all') ||
         Users.canDo(currentUser, 'alignment.sidebar')
   
     return (
       <React.Fragment>
         {shouldRenderSidebar && <SplitComponent name="SunshineSidebar" />}
-  
         <Components.HeadTags image={getSetting('siteImage')} />
         <TabNavigationMenu />
-  
         <RecommendationsAndCurated configName="frontpage" />
-        
         <HomeLatestPosts />
-  
-        <SingleColumnSection>
-          <SectionTitle title="Recent Discussion">
-            <div onClick={this.toggleShortformFeed}>
-              <SectionButton>
-                <AddBoxIcon />
-                New Shortform Post
-              </SectionButton>
-            </div>
-          </SectionTitle>
-          {showShortformFeed && <ShortformSubmitForm/>}
-          <RecentDiscussionThreadsList terms={{view: 'recentDiscussionThreadsList', limit:20}}/>
-        </SingleColumnSection>
+        <RecentDiscussionThreadsList terms={{view: 'recentDiscussionThreadsList', limit:20}}/>
```

Comment:
@Raemon This had copy-paste damage; you have to also apply the same change in `HomeEA.jsx`.